VCAC-A/README.md

```diff
@@ -3,28 +3,29 @@ Intel VCAC-A is designed to accelerate analytics computation. This README descri
 
 ## 1. Setup VCAC-A:
 
-Please follow the [Software Installation Guide, Section 2](https://cdrdv2.intel.com/v1/dl/getContent/611894) to build and configure the software packages for the host and the VCAC-A, with the following additional steps:    
+Please follow the [Software Installation Guide, Section 2.0-Section 2.2.4](https://cdrdv2.intel.com/v1/dl/getContent/611894) to build and configure the software packages for the host and the VCAC-A, with the following additional steps:    
 
 - **Install Docker Engine on Host and VCAC-A**
 
-Follow the [instructions](https://docs.docker.com/v17.09/engine/installation) to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.    
+Follow the [instructions](https://docs.docker.com/v17.09/engine/installation) to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.
 
-- **Install Intel OpenVINO on VCAC-A**
+Here is the script to install Docker on the VCAC-A:
```

Comment:
Here are the scripts to ...

Reply:
(1)  Please rename the scripts to be more uniformed:
setup-swarm.sh
upload-images.sh
upload-image1.sh
docker-run.sh
container-launcher.sh

(2) In all scripts, please add  $(env | grep -E '_(proxy)=' | sed 's/^/--build-arg /') to docker build, and --rm $(env | grep -E '_(proxy)=' | sed 's/^/-e /') to dock run commands.

(3) Some of the scripts use PROXY and some use proxy. Please use proxy uniformly.

Reply:
update as the comments. Please review again.